lib/virt_feature_test_base.pm

```diff
@@ -48,16 +48,21 @@ sub run_test {
     die('Please override this subroutine in children modules to run desired tests.');
 }
 
+sub try_remove_commands_logs {
+    # ignore the attempt timing out with "timeout 20" which exits before the
+    # script_run internal timeout
+    unless (defined(script_run("rm -f /root/{commands_history,commands_failure}"))) {
```

Comment:
`defined(script_run("rm -f /root/{commands_history,commands_failure}"))`

I am wondering in the case the ssh console is broken(which is our original attention), does this `script_run()` timeout? Does the test fail at once or continue with handling with `defined(script_run())`?

Reply:
Yes, this part I got wrong as `script_run` would always return a defined value unless it throws an exception on timeout. I will change it.